### PR TITLE
Coloring and optimization

### PR DESCRIPTION
**Pull request by Bulbang**
Added coloring and removed useless code

### CODE REVIEW

File: modules/front/classGameCell.js [hunk 1]
```diff
@@ -1,10 +1,29 @@
+const delay = 10;
+
 export class GameCell {
   constructor() {
     this.element;
     this.value;
     this.rowPos;
     this.colPos;
     this.status;
+    this.color = [
```
Comment (bogdanbryzh-official) [inline modules/front/classGameCell.js:10]: Why not hex?

File: yarn-error.log [hunk 2]
```diff
@@ -0,0 +1,46 @@
+Arguments:
```
Comment (bogdanbryzh-official) [inline yarn-error.log:1]: Delete this file 'git rm yarn-error.log' and add to .gitignore

File: modules/back/classGameMatrix.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
-import { getMaxRandom, getNewBase, isCellFree } from './base.js';
-import { size } from '../../game.config.js';
-import { gameCells } from './../../2048.js';
+import { getMaxRandom, getNewBase, isCellFree } from "./base.js";
```
Comment (bogdanbryzh-official) [inline modules/back/classGameMatrix.js:1]: Why " instead of '?

**Review by bogdanbryzh-official — CHANGES_REQUESTED** [review]

File: modules/front/classGameCell.js (hunk 1 above)
Comment (Bulbang) [reply modules/front/classGameCell.js:10]: Cuz I was manually iterating this colors in vs code)

File: modules/back/classGameMatrix.js (hunk 3 above)
Comment (Bulbang) [reply modules/back/classGameMatrix.js:1]: Don`t understand essence of the question

Comment (bogdanbryzh-official) [reply modules/back/classGameMatrix.js:1]: Single quotes are preferred for JS

File: .gitignore [hunk 4]
```diff
@@ -1 +1,2 @@
-.vercel
+yarn-error.log
```
Comment (bogdanbryzh-official) [inline .gitignore:1]: https://github.com/github/gitignore/blob/master/Node.gitignore

**Review by bogdanbryzh-official — CHANGES_REQUESTED** [review]

**Review by bogdanbryzh-official — APPROVED** [review]

**Pull request closed without merging**